Traceback (most recent call last):
  File "/Users/bkim/mambaforge/envs/myenv_x86/lib/python3.9/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/Users/bkim/mambaforge/envs/myenv_x86/lib/python3.9/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/bkim/mambaforge/envs/myenv_x86/lib/python3.9/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/bkim/mambaforge/envs/myenv_x86/lib/python3.9/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/Users/bkim/mambaforge/envs/myenv_x86/lib/python3.9/asyncio/base_events.py", line 647, in run_until_complete
    return future.result()
  File "/Users/bkim/mambaforge/envs/myenv_x86/lib/python3.9/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/Users/bkim/mambaforge/envs/myenv_x86/lib/python3.9/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Users/bkim/mambaforge/envs/myenv_x86/lib/python3.9/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
df_md.to_sql('md_wac_2015', conn)
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mValueError[0m                                Traceback (most recent call last)
Cell [0;32mIn[5], line 1[0m
[0;32m----> 1[0m [43mdf_md[49m[38;5;241;43m.[39;49m[43mto_sql[49m[43m([49m[38;5;124;43m'[39;49m[38;5;124;43mmd_wac_2015[39;49m[38;5;124;43m'[39;49m[43m,[49m[43m [49m[43mconn[49m[43m)[49m

File [0;32m~/mambaforge/envs/myenv_x86/lib/python3.9/site-packages/pandas/core/generic.py:2951[0m, in [0;36mNDFrame.to_sql[0;34m(self, name, con, schema, if_exists, index, index_label, chunksize, dtype, method)[0m
[1;32m   2794[0m [38;5;250m[39m[38;5;124;03m"""[39;00m
[1;32m   2795[0m [38;5;124;03mWrite records stored in a DataFrame to a SQL database.[39;00m
[1;32m   2796[0m 
[0;32m   (...)[0m
[1;32m   2947[0m [38;5;124;03m[(1,), (None,), (2,)][39;00m
[1;32m   2948[0m [38;5;124;03m"""[39;00m  [38;5;66;03m# noqa:E501[39;00m
[1;32m   2949[0m [38;5;28;01mfrom[39;00m [38;5;21;01mpandas[39;00m[38;5;21;01m.[39;00m[38;5;21;01mio[39;00m [38;5;28;01mimport[39;00m sql
[0;32m-> 2951[0m [38;5;28;01mreturn[39;00m [43msql[49m[38;5;241;43m.[39;49m[43mto_sql[49m[43m([49m
[1;32m   2952[0m [43m    [49m[38;5;28;43mself[39;49m[43m,[49m
[1;32m   2953[0m [43m    [49m[43mname[49m[43m,[49m
[1;32m   2954[0m [43m    [49m[43mcon[49m[43m,[49m
[1;32m   2955[0m [43m    [49m[43mschema[49m[38;5;241;43m=[39;49m[43mschema[49m[43m,[49m
[1;32m   2956[0m [43m    [49m[43mif_exists[49m[38;5;241;43m=[39;49m[43mif_exists[49m[43m,[49m
[1;32m   2957[0m [43m    [49m[43mindex[49m[38;5;241;43m=[39;49m[43mindex[49m[43m,[49m
[1;32m   2958[0m [43m    [49m[43mindex_label[49m[38;5;241;43m=[39;49m[43mindex_label[49m[43m,[49m
[1;32m   2959[0m [43m    [49m[43mchunksize[49m[38;5;241;43m=[39;49m[43mchunksize[49m[43m,[49m
[1;32m   2960[0m [43m    [49m[43mdtype[49m[38;5;241;43m=[39;49m[43mdtype[49m[43m,[49m
[1;32m   2961[0m [43m    [49m[43mmethod[49m[38;5;241;43m=[39;49m[43mmethod[49m[43m,[49m
[1;32m   2962[0m [43m[49m[43m)[49m

File [0;32m~/mambaforge/envs/myenv_x86/lib/python3.9/site-packages/pandas/io/sql.py:697[0m, in [0;36mto_sql[0;34m(frame, name, con, schema, if_exists, index, index_label, chunksize, dtype, method, engine, **engine_kwargs)[0m
[1;32m    692[0m [38;5;28;01melif[39;00m [38;5;129;01mnot[39;00m [38;5;28misinstance[39m(frame, DataFrame):
[1;32m    693[0m     [38;5;28;01mraise[39;00m [38;5;167;01mNotImplementedError[39;00m(
[1;32m    694[0m         [38;5;124m"[39m[38;5;124m'[39m[38;5;124mframe[39m[38;5;124m'[39m[38;5;124m argument should be either a Series or a DataFrame[39m[38;5;124m"[39m
[1;32m    695[0m     )
[0;32m--> 697[0m [38;5;28;01mreturn[39;00m [43mpandas_sql[49m[38;5;241;43m.[39;49m[43mto_sql[49m[43m([49m
[1;32m    698[0m [43m    [49m[43mframe[49m[43m,[49m
[1;32m    699[0m [43m    [49m[43mname[49m[43m,[49m
[1;32m    700[0m [43m    [49m[43mif_exists[49m[38;5;241;43m=[39;49m[43mif_exists[49m[43m,[49m
[1;32m    701[0m [43m    [49m[43mindex[49m[38;5;241;43m=[39;49m[43mindex[49m[43m,[49m
[1;32m    702[0m [43m    [49m[43mindex_label[49m[38;5;241;43m=[39;49m[43mindex_label[49m[43m,[49m
[1;32m    703[0m [43m    [49m[43mschema[49m[38;5;241;43m=[39;49m[43mschema[49m[43m,[49m
[1;32m    704[0m [43m    [49m[43mchunksize[49m[38;5;241;43m=[39;49m[43mchunksize[49m[43m,[49m
[1;32m    705[0m [43m    [49m[43mdtype[49m[38;5;241;43m=[39;49m[43mdtype[49m[43m,[49m
[1;32m    706[0m [43m    [49m[43mmethod[49m[38;5;241;43m=[39;49m[43mmethod[49m[43m,[49m
[1;32m    707[0m [43m    [49m[43mengine[49m[38;5;241;43m=[39;49m[43mengine[49m[43m,[49m
[1;32m    708[0m [43m    [49m[38;5;241;43m*[39;49m[38;5;241;43m*[39;49m[43mengine_kwargs[49m[43m,[49m
[1;32m    709[0m [43m[49m[43m)[49m

File [0;32m~/mambaforge/envs/myenv_x86/lib/python3.9/site-packages/pandas/io/sql.py:2189[0m, in [0;36mSQLiteDatabase.to_sql[0;34m(self, frame, name, if_exists, index, index_label, schema, chunksize, dtype, method, **kwargs)[0m
[1;32m   2178[0m             [38;5;28;01mraise[39;00m [38;5;167;01mValueError[39;00m([38;5;124mf[39m[38;5;124m"[39m[38;5;132;01m{[39;00mcol[38;5;132;01m}[39;00m[38;5;124m ([39m[38;5;132;01m{[39;00mmy_type[38;5;132;01m}[39;00m[38;5;124m) not a string[39m[38;5;124m"[39m)
[1;32m   2180[0m table [38;5;241m=[39m SQLiteTable(
[1;32m   2181[0m     name,
[1;32m   2182[0m     [38;5;28mself[39m,
[0;32m   (...)[0m
[1;32m   2187[0m     dtype[38;5;241m=[39mdtype,
[1;32m   2188[0m )
[0;32m-> 2189[0m [43mtable[49m[38;5;241;43m.[39;49m[43mcreate[49m[43m([49m[43m)[49m
[1;32m   2190[0m [38;5;28;01mreturn[39;00m table[38;5;241m.[39minsert(chunksize, method)

File [0;32m~/mambaforge/envs/myenv_x86/lib/python3.9/site-packages/pandas/io/sql.py:833[0m, in [0;36mSQLTable.create[0;34m(self)[0m
[1;32m    831[0m [38;5;28;01mif[39;00m [38;5;28mself[39m[38;5;241m.[39mexists():
[1;32m    832[0m     [38;5;28;01mif[39;00m [38;5;28mself[39m[38;5;241m.[39mif_exists [38;5;241m==[39m [38;5;124m"[39m[38;5;124mfail[39m[38;5;124m"[39m:
[0;32m--> 833[0m         [38;5;28;01mraise[39;00m [38;5;167;01mValueError[39;00m([38;5;124mf[39m[38;5;124m"[39m[38;5;124mTable [39m[38;5;124m'[39m[38;5;132;01m{[39;00m[38;5;28mself[39m[38;5;241m.[39mname[38;5;132;01m}[39;00m[38;5;124m'[39m[38;5;124m already exists.[39m[38;5;124m"[39m)
[1;32m    834[0m     [38;5;28;01melif[39;00m [38;5;28mself[39m[38;5;241m.[39mif_exists [38;5;241m==[39m [38;5;124m"[39m[38;5;124mreplace[39m[38;5;124m"[39m:
[1;32m    835[0m         [38;5;28mself[39m[38;5;241m.[39mpd_sql[38;5;241m.[39mdrop_table([38;5;28mself[39m[38;5;241m.[39mname, [38;5;28mself[39m[38;5;241m.[39mschema)

[0;31mValueError[0m: Table 'md_wac_2015' already exists.
ValueError: Table 'md_wac_2015' already exists.

